service/src/main/java/uk/gov/hmcts/reform/fpl/config/GeneralInboxLookupConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.fpl.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+@lombok.Data
+@Service
+public class GeneralInboxLookupConfiguration {
+    private final String generalInbox;
```

Comment:
Why not just inbox?

Reply:
changed to `PublicLawEmailLookupConfiguration`